### DISCUSSION TITLE
SERIOUS problems with STUDENT BENEFITS

### DISCUSSION THREAD

**Discussion by Rxxbertx** [community/community#58645] [GitHub Education]
I am very tired of the management that GitHub has to grant student benefits...THE FIRST TIME a year ago I had to apply 3 TIMES to be admitted, now after a year..I have been rejected 3 TIMES and they keep rejecting me.
In Spain especially in Castilla y Leon many schools do not have their own email, THING THAT GITHUB DOES NOT UNDERSTAND AND whoever is the employee in charge of managing that information also does not want to pay attention and take it seriously...I have repeated several times that the email is @educa.jcyl.es AND they keep rejecting the application because of the EMAIL.
There is also a big problem with the documentation to prove that you are a student.... I SENT PICTURE OF MY ID CARD (rejected), PICTURE OF THE WEB PAGE OF THE CENTER WHERE THE TEACHERS SEND HOMEWORK AND THINGS OF THAT STYLE (rejected, and that a colleague of mine WAS ACCEPTED only with that), PICTURE OF A DOCUMENT CERTIFIED BY THE CENTER that I am studying a degree in development of multiplatform applications (rejected) WHAT ELSE CAN I SEND YOU? I am very angry and upset about all this, considering that I have already been accepted a year ago...and now I am simply NOT accepted. And that I use Jetbrains products, GitHub copilot, git Kraken....(It should be noted that Jetbrains accepted me at the first time without having GitHub student, they were very kind) PLEASE GITHUB review my application and read what I write in them.... And please accept 🥹...

**Comment by samus-aran** [comment]
Hi @Rxxbertx 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

Ok, lets see where we can help. 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
